### DISCUSSION TITLE
Can't convert openLLAMA 7b

### DISCUSSION THREAD

**Discussion by Darin755** [ggml-org/llama.cpp#2791] [Q&A]
Hello,

I am trying to convert openLLAMA 7b using the convert script. I ran the command in order to convert:

```python3 convert.py ../open_llama_7b_v2/``` (open_llama_7b_v2 is the name of the directory with openllama)

However, I get the following error:

```ValueError: unknown format: ../open_llama_7b_v2/pytorch_model-00001-of-00002.bin```

I don't know how to troubleshoot this. Does anyone else have more information?

**Comment by Yeetster-cmd** [comment]
I don't think convert.py itself works with the new version of llama.cpp, but there are many other convert files, the one I think may work (though I'm not sure) may be `convert-llama-hf-to-gguf.py`.

**Reply by Darin755** [reply]
That script just gives me a serialization error.

**Reply by KerfuffleV2** [reply]
`convert.py` works with the latest version. I'm pretty sure it is the recommended tool for model conversion (at least LLaMA) models.

OP, "unknown format" sounds like the file is corrupted or something. Double check the validity of the files. If you got it from HuggingFace, HF will show you the SHA256 for each file. On Linux (maybe Mac?) you can run the `sha256sum` tool. Not sure how to check hashes on Windows.

**Comment by bukosabino** [comment]
Hi @Darin755 ,

Please, check [this comment](https://github.com/ggerganov/llama.cpp/issues/1994#issuecomment-1662714838).

It was the solution for me.
